docs/details/widgets/bar.rst

```diff
@@ -37,8 +37,8 @@ Usage
 Orientation and size
 --------------------
 
-- for orientation, width and height, simply set width and height properties;
-- :cpp:expr:`lv_bar_set_orientation(bar, orientation)` to override orientation
+- for orientation, width and height, simply set width and height style properties;
+- :cpp:expr:`lv_bar_set_orientation(slider, orientation)` to override orientation
```

Comment:
I think this should stay as `bar`

Reply:
Wow -- must have been a copy/paste casualty.  Good catch!